src/lib389/lib389/backend.py

```diff
@@ -607,6 +608,125 @@ def _lint_cl_trimming(self):
             self._log.debug(f"_lint_cl_trimming - backend ({suffix}) is not replicated")
             pass
 
+    def _lint_system_indexes(self):
+        """Check that system indexes are correctly configured"""
+        bename = self.lint_uid()
+        suffix = self.get_attr_val_utf8('nsslapd-suffix')
+        indexes = self.get_indexes()
+
+        # Default system indexes taken from ldap/servers/slapd/back-ldbm/instance.c
+        expected_system_indexes = {
+            'entryrdn': {'types': ['subtree'], 'matching_rule': None},
+            'parentId': {'types': ['eq'], 'matching_rule': 'integerOrderingMatch'},
+            'objectClass': {'types': ['eq'], 'matching_rule': None},
+            'aci': {'types': ['pres'], 'matching_rule': None},
+            'nsUniqueId': {'types': ['eq'], 'matching_rule': None},
+            'nsds5ReplConflict': {'types': ['eq', 'pres'], 'matching_rule': None}
```

Comment:
Any reason why 'nscpEntryDN' and 'ancestorid' are not listed ?

Reply:
Initially I based my list on default system indexes (`nssystemindex: true`) from `ldap/ldif/template-dse.ldif.in`. I see now that I also missed:
- nsCertSubjectDN
- numsubordinates
- nsTombstoneCSN
- targetuniqueid